Jenkinsfile

```diff
@@ -4,11 +4,6 @@ pipeline {
     }
 
     stages {
-        stage('Check for unused asm') {
-            steps {
-                sh './tools/find_unused_asm.sh'
-            }
-        }
```

Comment:
While this is removed, what was going to be the plan going forward for handling asm in other versions? Is the plan to go forward with transient asm?

Reply:
I'm not sure yet, but if we do commit asm again, we can always add a similar check back. This check serves no purpose now though

Reply:
Yeah we're going with transient asm, though it's not 100% decided I don't see a path where transient asm can't be made to work